try {
    // Código que puede lanzar una excepción
} catch (IOException e) {
    System.out.println("Se produjo un error de IO");
} finally {
    System.out.println("Este bloque siempre se ejecutará");
}
